### ISSUE TITLE
LazyImageView processors are not applied to request

### ISSUE THREAD

**Issue by francuim-d** [kean/Nuke#633]
Firstly, thank u for your framework!
I noticed that when LazyImageView has default processors and then the url of the image is assigned to start the download, these processors are not applied to the created request.
I guess there's a typo in here. Request's processors should be empty to apply default processors.
```swift
if let processors = self.processors, !processors.isEmpty, !request.processors.isEmpty {
    request.processors = processors
}
```

**Comment by kean** [comment]
Yes, it's a typo/bug.

**Comment by kean** [comment]
Fixed in [Nuke 11.5.0](https://github.com/kean/Nuke/releases/tag/11.5.0).
